src/tutorials/0004/07.vue

```diff
@@ -52,7 +52,7 @@ const validate = async (result, ipfs) => {
 
   // Check whether array contains certain directory hash & certain directory name
   function contains (certainHash, certainName) {
-    if (!result.error) {
+    if (Array.isArray(result)) {
```

Comment:
Doesn't this need to be after the conditional for there being no result so that we don't run into an error about it being undefined?

Reply:
This is just the function declaration, we are only using the `contains` function after checking if the result is not undefined.